index.html

```diff
@@ -160,13 +178,13 @@ <h1 class="section-header" id="my-name">Get started</h1>
             <textarea class="border-radius" placeholder="Enter your message here" maxlength="500" required rows="6"></textarea>
           </li>
           <li class="form-items">
-            <button type="submit" class=""> Start collaboration </button>
+            <button type="submit" class="my-button"> Start collaboration </button>
```

Comment:
- [ ] Kindly make all the changes requested by the previous reviewer. You button still do not have a hover effect and it still has a horizontal scroll.
  https://www.loom.com/share/965f33c3610945d698ae2ba59531f7bc